### PR TITLE
Ukrainian localization is added

### PR DESCRIPTION
**Pull request by recluse27**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Why is this valuable (or not)?
Ukrainian language is added

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by davorpa** [comment]
Please, remove crosslinks from files that dont are your translation.

File: HOWTO-fr.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
+N'hésitez pas à poser des questions; chaque contributeur a commencé par un premier PR. Vous pourriez être notre millième!
 
-Même si vous êtes un contributeur open source expérimenté, il y a des choses qui peuvent vous faire trébucher. Une fois que vous avez soumis votre PR, GitHub Actions exécutera un linter, trouvant souvent de petits problèmes d'espacement ou d'alphabétisation. Si vous obtenez un bouton vert, tout est prêt pour l'examen, mais sinon, cliquez sur "Détails" sous la vérification qui n'a pas réussi pour découvrir ce que le linter n'a pas aimé. Résolvez le problème et ajoutez un commit à votre PR.
+Même si vous êtes un contributeur open source expérimenté, il y a des choses qui peuvent vous trébucher. Une fois que vous avez soumis votre PR, GitHub Actions exécutera un linter, trouvant souvent de petits problèmes d'espacement ou d'alphabétisation. Si vous obtenez un bouton vert, tout est prêt pour l'examen, mais sinon, cliquez sur "Détails" sous la vérification qui n'a pas réussi à découvrir ce que le linter n'a pas aimé. Résolvez le problème et ajoutez un commit à votre PR.
```
Comment (davorpa) [inline HOWTO-fr.md:13]: Why?

**Comment by davorpa** [comment]
Locale for Ukrainian is `uk` https://www.localeplanet.com/icu/uk-UA/index.html

**Review by recluse27 — COMMENTED** [review]
@davorpa Fixes were made

**Comment by eshellman** [comment]
please use just -uk , not -uk_UA

File: HOWTO-ar.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-إقرأ هذا بلغات أخرى: [English](HOWTO.md), [Français](HOWTO-fr.md), [Español](HOWTO-es.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md), [Русский](HOWTO-ru.md), [Deutsch](HOWTO-de.md), [Bahasa Indonesia](HOWTO-id.md), [Українська](HOWTO-uk_UA.md).
+إقرأ هذا بلغات أخرى: [English](HOWTO.md), [Français](HOWTO-fr.md), [Español](HOWTO-es.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md), [Русский](HOWTO-ru.md), [Deutsch](HOWTO-de.md), [Bahasa Indonesia](HOWTO-id.md), [Українська](HOWTO-uk.md).
```
Comment (davorpa) [inline HOWTO-ar.md:1]: With the intention of avoiding merge conflicts in other pending PRs, I'm in favour of avoid this kind of crosslinks in all changed files. **Try to scope to ​​`-uk`** files.

**Review by davorpa — CHANGES_REQUESTED** [review]

File: HOWTO-ar.md (hunk 2 above)
Comment (recluse27) [reply HOWTO-ar.md:1]: @davorpa In any case you will need to add those references in other files. Wouldn't it be better to do it right now?

**Comment by recluse27** [comment]
@eshellman files were renamed

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**